apps/example/e2e/src/usecases/OffRamps.js

```diff
@@ -69,6 +69,10 @@ export default offRamps = () => {
 
         await waitForElementById('FiatExchangeInput')
         await element(by.id('FiatExchangeInput')).replaceText('2')
+        // Hide the keyboard on Android
```

Comment:
it'd reduce complexity if we could minimise separate code paths for the different platforms. could we do a tap on the screen to dismiss the keyboard instead? (typing "\n" seems like an unnatural user path)